pkgs/top-level/python-packages.nix

```diff
@@ -571,6 +571,8 @@ in {
 
   apcaccess = callPackage ../development/python-modules/apcaccess { };
 
+  apc-temp-fetch = callPackage ../development/python-modules/apc-temp-fetch { };
```

Comment:
If it is not used as a dependency in other packages we don't need or want this entry to be more free what we do with the package.